source/fonts/__init__.py

```diff
@@ -3,38 +3,70 @@
 # Copyright (C) 2019 NV Access Limited
 # This file is covered by the GNU General Public License.
 # See the file COPYING for more details.
+from typing import List
 
 from logHandler import log
 import os
+from ctypes import WinDLL
+gdi32 = WinDLL("gdi32.dll")
 """
 Loads custom fonts for use in NVDA.
 """
 
+fontsDir = os.path.abspath("fonts")
+
 
 def _isSupportedFontPath(f: str) -> bool:
 	return os.path.isfile(f) and (
-		f.endswith("otf")
-		or f.endswith("ttf")
+		f.endswith(".otf")
+		or f.endswith(".ttf")

```

Comment:
I think endswith supports a tuple on which it will automatically do an `or`

Reply:
Given this won't have any impact on behavior, I'd prefer to just get this merged.